src/frontend/Felles/Personopplysninger/Adressehistorikk.tsx

```diff
@@ -27,6 +21,18 @@ const FetTekst = styled.span`
 
 const MAX_LENGDE_ADRESSER = 5;
 
+const TekstMedTagWrapper = styled.div`
```

Comment:
```suggestion
const FlexBox = styled.div`
```
Evt. `FlexBoxCentered` `CenteredFlexBox`